backend/src/main/java/com/daedan/festabook/place/controller/PlaceController.java

```diff
@@ -75,10 +75,10 @@ public PlacePreviewResponses getAllPreviewPlaceByFestivalId(
     @ApiResponses(value = {
             @ApiResponse(responseCode = "200", useReturnTypeSchema = true),
     })
-    public PlaceResponse getPlaceWithDetailByPlaceId(
+    public PlaceResponse getPlaceById(
             @PathVariable Long placeId
     ) {
-        return placeService.getPlaceWithDetailByPlaceId(placeId);
+        return placeService.getPlaceById(placeId);
```

Comment:
[컨벤션] By Id 에 구체적인 명칭 정의하기로 했습니다.

```suggestion
        return placeService.getPlaceByPlaceId(placeId);
```

Reply:
```java
    // TODO: ExceptionHandler 등록 후 예외 변경
    private Place getPlaceByPlaceId(Long placeId) {
        return placeJpaRepository.findById(placeId)
                .orElseThrow(() -> new BusinessException("존재하지 않는 플레이스입니다.", HttpStatus.NOT_FOUND));
    }
```

아래 있는 메서드와 시그니처가 동일해서 불가능합니다.